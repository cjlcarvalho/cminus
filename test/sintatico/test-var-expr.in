void main(void)
{
    x * y - 4 / z + w * (k + 2) + j[10] - j[x] - j[x-2];
}
