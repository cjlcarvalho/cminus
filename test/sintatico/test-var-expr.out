[program 
  [fun-declaration 
    [void]
    [main]
    [params]
    [compound-stmt 
      [- 
        [- 
          [+ 
            [+ 
              [- 
                [* [var [x]] [var [y]]] 
                [/ [4] [var [z]]]] 
              [* [var [w]] 
                [+ [var [k]] [2]]]] [var [j] [10]]] [var [j] [var [x]]]] [var [j] 
          [- [var [x]] [2]]]] 
    ]
  ]
]
